peps/pep-0740.rst

```diff
@@ -6,14 +6,17 @@ Author: William Woodruff <william@yossarian.net>,
 Sponsor: Donald Stufft <donald@stufft.io>
 PEP-Delegate: Donald Stufft <donald@stufft.io>
 Discussions-To: https://discuss.python.org/t/pep-740-index-support-for-digital-attestations/44498
-Status: Provisional
+Status: Final
 Type: Standards Track
 Topic: Packaging
 Created: 08-Jan-2024
 Post-History: `02-Jan-2024 <https://discuss.python.org/t/pre-pep-exposing-trusted-publisher-provenance-on-pypi/42337>`__,
               `29-Jan-2024 <https://discuss.python.org/t/pep-740-index-support-for-digital-attestations/44498>`__
-Resolution: https://discuss.python.org/t/pep-740-index-support-for-digital-attestations/44498/26
+Resolution: `17-Jul-2024 <https://discuss.python.org/t/pep-740-index-support-for-digital-attestations/44498/26>`__
 
+.. canonical-pypa-spec:: :ref:`packaging:index-hosted-attestations`
+
+.. canonical-doc:: `PyPI - Digital Attestations <https://docs.pypi.org/attestations/>`__
```

Comment:
NB: I've left both the spec and doc links in, since the spec is useful to implementors and the doc is useful to consumers. But I'm happy to remove the doc link if convention is to only link the PyPA spec.